workflows/VGP-assembly-v2/VGP-Decontamination/VGP-Decontamination-pipeline---Db-not-included-tests.yml

```diff
@@ -0,0 +1,20 @@
+- doc: Test outline for VGP-Decontamination-pipeline---Db-not-included
+  job:
+    Scaffolded assembly (fasta):
```

Comment:
Scaffolded assembly should be sufficient, the user interface should tell you it is fasta (and it seems fasta.gz also works). You;ll have to rename the parameter in the workflow too.